framework/common/test/encryption_test.cpp

```diff
@@ -25,6 +25,17 @@ static bool check(int &errors, bool check, std::string message)
     return check;
 }
 
+std::string dump_buffer(uint8_t *buffer, size_t len)
```

Comment:
Could we promote beerocks_utils::dump_buffer to somewhere in framework, so it can be used here?

Reply:
I forgot to remove this, we don't need to dump the buffer in the test. In any case we can - perhaps @CoralMalachi can do that?

Reply:
Yes. @tomereli

Reply:
@CoralMalachi already done, don't worry about it :)